rita_common/src/usage_tracker/mod.rs

```diff
@@ -480,6 +482,18 @@ pub fn get_payments_data() -> VecDeque<PaymentHour> {
     usage_tracker_var.payments.clone()
 }
 
+/// Gets ops last seen hour for this router, stored on the local disk at periodic intervals
+pub fn get_last_seen_hour() -> u64 {
+    let usage_tracker_var = &*(USAGE_TRACKER.read().unwrap());
+    usage_tracker_var.ops_last_seen
+}
+
+/// Update the ops last seen hour for the last hour we've confirmed has been received by ops
+pub fn update_last_seen(last_seen_hour: u64) {
```

Comment:
Right now ops last seen is saved to the disk because it's part of usage tracker and by default it's 0, meaning we send all data.

I don't really see the need for this value to be here, it could be loop state initialized in update_loop.rs and updated as a return value from the operator update function. it would reset to None every router reboot, then we would send data on the second checkin (the first checkin we would send no data because we do not yet know what ops has seen)

If we store the ops last seen value here it will be saved to the disk and that becomes much harder to reason about.

1) what happens if this value is updated, but somehow ops doesn't have the data (failed to save, database roll back, some issue on ops side). We would be unable to get that data without another Rita update
2) what happens if this value is larger than the current router time (remember routers have unreliable clocks, there should be an underflow check somewhere so we don't panic) 
3) If this value is just junk how does it become re-syncronized with ops? If it's on disk this is complex, if it's in memory it's guaranteed to become re-syncronized every checkin.